src/components/ContentUploader/UploadStateContent.js

```diff
@@ -17,7 +17,11 @@ const UploadStateContent = ({ message, inputLabel, useButton = false, onChange }
     const messageContent = message ? <div className='bcu-upload-state-message'>{message}</div> : null;
     const inputLabelClass = useButton ? 'btn btn-primary be-input-btn' : 'be-input-link';
 
-    const handleChange = (event: Event<*>) => {
+    const handleChange = (event: Event) => {
+        if (!onChange) {
+            return;
+        }
+
         onChange(event);
```

Comment:
This will be a react synthetic event. Do we want that or the original event?

Reply:
btw dunno if you saw https://github.com/box/box-ui-elements/pull/297/files#r186877887

Reply:
yeah Jeremy pointed to me your comment. updated